src/replicache-mutation-recovery.test.ts

```diff
@@ -86,6 +88,24 @@ async function createAndPersistClientWithPendingLocal(
   return localMetas;
 }
 
+function createPushBody(
+  client1ID: string,
```

Comment:
s/client1ID/clientID/

Reply:
Done: https://github.com/rocicorp/replicache/pull/825

thanks for the review